ocs2_robotic_examples/ocs2_legged_robot_ros/launch/legged_robot_ddp.launch.py

```diff
@@ -76,6 +76,12 @@ def generate_launch_description():
             output='screen',
             prefix=["gnome-terminal --"],
         ),
+        Node(
+            name='joy_node',
+            package='joy',
+            executable='joy_node',
+            output='screen',
+        ),
```

Comment:
Just checking, does this behave properly if I don't have a joystick plugged in? Or should it be gated behind a `joystick:=true` launch parameter?

Reply:
if a joystick is not attached then nothing happens, as soon as it gets plugged in then it publishes the appropriate commands